src/components/chat-bubble/ChatBubble.scss

```diff
@@ -122,5 +122,7 @@
 	&-photo {
 		height: 30rem;
 		width: auto;
+		min-width: 100%;
```

Comment:
ensures the photo to fill chat bubble in case if the text is longer than the photo